### PR TITLE
Don't use the deprecated method

### PR DESCRIPTION
**Pull request by nickvergessen**
Heavily reduces the warning log spam on pages like activity which before printed a warning for each activity every 30 seconds

### CODE REVIEW

**Review by PVince81 — APPROVED** [review]
:+1:

strange that we don't have a new utility function for that though

**Review by artonge — APPROVED** [review]

**Comment by nickvergessen** [comment]
> strange that we don't have a new utility function for that though

We have, and it's marked as deprecated. Also fine with reverting the deprecation:
https://github.com/nextcloud/server/blob/3e624dc39966f54c6d418eee2e8141fc5c21f751/core/src/OC/util.js#L129-L142

File: core/src/init.js [hunk 1]
```diff
@@ -98,7 +98,16 @@ const initLiveTimestamps = () => {
...
 	setInterval(() => {
 		$('.live-relative-timestamp').each(function() {
-			$(this).text(OC.Util.relativeModifiedDate(parseInt($(this).attr('data-timestamp'), 10)))
+			const timestamp = parseInt($(this).attr('data-timestamp'), 10)
+			const diff = moment().diff(moment(timestamp))
+			let relativeTime = ''
+			if (diff >= 0 && diff < 45000) {
+				relativeTime = t('core', 'seconds ago')
+			} else {
+				relativeTime = moment(timestamp).fromNow()
```
Comment (juliushaertl) [inline core/src/init.js:107]: If using `@@nextcloud/moment` this sould already be the case when just calling fromow():

https://github.com/nextcloud/nextcloud-moment/blob/master/test/index.test.js#L18

**Review by juliusknorr — CHANGES_REQUESTED** [review]

File: core/src/init.js (hunk 1 above)
Comment (juliushaertl) [reply core/src/init.js:107]: image

Which is what gets overwritten in https://github.com/nextcloud/nextcloud-moment/blob/master/lib/index.ts#L22

Comment (nickvergessen) [reply core/src/init.js:107]: So actually we should split by the time diff < 0 and also use tonow?

Comment (juliushaertl) [reply core/src/init.js:107]: No, moment seems to handle that just fine with only calling fromNow:

```
moment(Date.now()-10).fromNow()
"vor ein paar Sekunden"
moment(Date.now()+10).fromNow()
"in ein paar Sekunden"
```

Comment (nickvergessen) [reply core/src/init.js]: Finally fixed

**Review by juliusknorr — APPROVED** [review]

**Pull request merged**